multiversx_sdk_core/transaction_factories/transactions_factory_config.py

```diff
@@ -35,3 +35,8 @@ def __init__(self, chain_id: str) -> None:
         self.gas_limit_delegation_operations = 1_000_000
         self.additional_gas_limit_per_validator_node = 6_000_000
         self.additional_gas_for_delegation_operations = 10_000_000
+
+        # Configuration for token transfers
+        self.esdt_transfer = 200_000
```

Comment:
add a `_gas` suffix to all these configs

Reply:
added a `gas_limit_...` prefix to ensure consistency with the other names.